df.world-site.xml

```diff
@@ -431,7 +431,14 @@
             <static-array count='51' pointer-type='site_realization_crossroads'/>
         </static-array>
         <static-array name='river_map' count='51'>
-            <static-array count='51' type-name='pointer'/>
+            <pointer>
+                <static-array name='start' type-name='int32_t' count='48'/>
+                <static-array name='end'   type-name='int32_t' count='48'/>
+                <int32_t name="unk_180"/>
+                <int32_t name="unk_184"/>
+                <int32_t name="start_array_limits"/>
```

Comment:
Yes, you are right

This is unknown struct used by DF to draw the rivers of a site when it's exported as a bitmap.

start array tells the start pixel where a river must be draw until its corresponding value in end array.

These pixels, of course are local to the subdivision that river_map[][] does of the 816 x 816 pixels that a image of a world site is.